controllers/configurationpolicy_utils.go

```diff
@@ -53,7 +54,7 @@ func addRelatedObjects(
 		}
 
 		relatedObject.Object.APIVersion = rsrc.GroupVersion().String()
-		relatedObject.Object.Kind = rsrc.Resource
+		relatedObject.Object.Kind = kind
```

Comment:
What's the purpose of this change? I feel like I'm missing something... was it recording incorrect information before?

Reply:
@JustinKuli Sort of - the `resource` and `kind` for an object are pretty similar, but I found that passing the resource in instead of the kind would cause the mapping function to be unable to query the object. For example, for pods the resource is `pods` and the kind is `Pod`. Using the kind makes the relatedObjects array line up better with the actual object (which displays 'kind') and makes the code for object deletion cleaner.